### PR TITLE
docs: Starknet state-Edits and clarifications

### PR DESCRIPTION
**Pull request by stoobie**
### Description of the Changes

Tightening up the info in Starknet state, also updating for v3 transactions.

### PR Preview URL

https://starknet-io.github.io/starknet-docs/pr-943/documentation/architecture_and_concepts/Network_Architecture/starknet-state/

### Check List

- [x] Changes have been done against dev branch, and PR does not conflict
- [x] PR title follows the convention: `<docs/feat/fix/chore>(optional scope): <description>`, e.g: `fix: minor typos in code`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starknet-io/starknet-docs/943)

### CODE REVIEW

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet-state.adoc [hunk 1]
```diff
@@ -3,36 +3,36 @@
 
 :stem: latexmath
 
-The state of Starknet consists of:
+Starknet's state consists of:
 
 * xref:architecture_and_concepts:Smart_Contracts/contract-classes.adoc[Contract classes]: a mapping
 between class hash and the class definition
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet-state.adoc:9]:
```suggestion
between the class hash and the class definition
```

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet-state.adoc [hunk 2]
```diff
@@ -3,36 +3,36 @@
...
 
-A contract state consists of:
+A contract instance's state consists of:
 
 * xref:Smart_Contracts/class-hash.adoc[Class hash]: defines the functionality of the contract
 * xref:Smart_Contracts/contract-storage.adoc[Contract storage]: a key-value mapping where the key/values are field elements
 * xref:Accounts/approach.adoc#replay_protection[Contract nonce]: the number of transactions sent from this contract
 
-With the above definition, we can provide a brief sketch of Starknet's transition function. A transaction stem:[$tx$] transitions the system from state stem:[$S$] to state stem:[$S'$] if:
+With the above definition, we can provide a brief outline of Starknet's transition function. A transaction stem:[$tx$] transitions the system from state stem:[$S$] to state stem:[$S'$] if:
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet-state.adoc:18]:
```suggestion
[#transitioning_to_a_new_state]
== Transitioning to a new state

A transaction stem:[$tx$] transitions the system from state stem:[$S$] to state stem:[$S'$] if:
```

File: components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet-state.adoc [hunk 3]
```diff
@@ -3,36 +3,36 @@
...
-contract
-address, and the specific entry point are part of the transaction)
-* stem:[$tx$] is a xref:Network_Architecture/transactions.adoc#deploy_account_transaction[deploy_account] transaction, and stem:[$S'$] contains the new contract's state at the contract's address. Additionally, the storage of stem:[$S$] is updated
-according to the execution of the contract's constructor.
-* stem:[$tx$] is a xref:Network_Architecture/transactions.adoc#declare-transaction[declare] transaction, and stem:[$S'$] contains the class hash and definition in the contract classes mapping
+* stem:[$tx$] is an xref:Network_Architecture/transactions.adoc#invoke_transaction[invoke] transaction, and the storage of stem:[$S'$] is the result of executing the target contract code with respect to the previous state stem:[$S$]. The arguments,
+contract instance's address, and the specific function entry point are part of the transaction.
+* stem:[$tx$] is a xref:Network_Architecture/transactions.adoc#deploy_account_transaction[deploy_account] transaction, and stem:[$S'$] contains the new contract instance's state at the contract instance's address. Additionally, the storage of stem:[$S$] is updated
```
Comment (stoobie) [inline components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/starknet-state.adoc:22]:
```suggestion
* stem:[$tx$] is a xref:Network_Architecture/transactions.adoc#deploy_account_transaction[deploy account] transaction, and stem:[$S'$] contains the new contract instance's state at the contract instance's address. Additionally, the storage of stem:[$S$] is updated
```

**Comment by LucasLvy** [comment]
Screenshot 2023-11-21 at 10 19 49
any chance to make this a table or smth where we can see better the separation ?

**Comment by stoobie** [comment]
> Screenshot 2023-11-21 at 10 19 49 any chance to make this a table or smth where we can see better the separation ?
@LucasLvy Take a look now. I added stripes to make it clearer.

Is this the only issue you? Other than that is it OK to merge?

**Comment by LucasLvy** [comment]
yes

**Pull request merged**